verification/curator-service/ui/README.md

```diff
@@ -34,17 +50,34 @@ US).
 - `admin3`: Optional. Meaning differs by country (e.g., this is municipality in the
 US).
 [See full definition here](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).
-- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or `Other`.
-- `ageRangeStart`: Optional. Number (years) from 0 to 120.
-- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+
+##### Events
 - `dateConfirmed`: **Required**. Date (MM/DD/YYYY) on which the case was
 confirmed.
+- `confirmationMethod`: Optional. Mechanism by which the case was confirmed
+(e.g., `PCR test`).
 - `hospitalized`: Optional. One of `true` or `false`.
 - `dateHospitalized`: Optional. Date (MM/DD/YYYY) on which the patient was
 hospitalized.
+- `icuAdmission`: Optional. One of `true` or `false`.
+- `dateIcuAdmission`: Optional. Date (MM/DD/YYYY) on which the patient was
+admitted to the ICU.
 - `outcome`: Optional. One of either `Death` or `Recovered`.
 - `dateOutcome`: Optional. Date (MM/DD/YYYY) on which the above outcome
 transpired.
+- `dateSymptomOnset`: Optional. Date (MM/DD/YYYY) on which symptoms first
```

Comment:
There are 2 more event dates in the single case flow: firstClinicalConsultationDate and selfIsolationDate

Reply:
Yep, for now I think we're just including the fields that we've identified to exist in sources we'd plausibly bulk upload. Stephen made a list in #646 which I'm working off of here.

Reply:
Hm interesting... I would think we would want to allow uploading all linelist data using bulk upload. Sources could always add data or new sources could crop up. Not sure why we'd want to restrict it. (Happy for another PR but let's keep an issue open to track)

Reply:
SG, I've opened #718.